### PR TITLE
LTD-1667-PDF

### PR DESCRIPTION
**Pull request by seijihg**
https://uktrade.atlassian.net/browse/LTD-1667

### CODE REVIEW

**Review by depsiatwal — COMMENTED** [review]
no tests cover this ?

File: api/cases/generated_documents/signing.py [hunk 1]
```diff
@@ -49,13 +49,22 @@ def _get_signature_text(date):
 
 
 def _add_blank_page(pdf_bytes: bytes):
+
+    orignal_pdf_buffer = BytesIO(pdf_bytes)
```
Comment (hnryjmes) [inline api/cases/generated_documents/signing.py:53]: spelling; you could change this to `original_pdf_buffer` to be consistent

**Review by hnryjmes — APPROVED** [review]

File: api/cases/generated_documents/tests/test_generate_document.py [hunk 2]
```diff
@@ -534,6 +535,8 @@ def test_pdf_titles(self, temp, title):
         case = self.create_standard_application_case(self.organisation, user=self.exporter_user)
         html = generate_preview(layout=temp, case=case, text="")
         pdf = html_to_pdf(html, temp, None)
+        pdf = sign_pdf(pdf)
```
Comment (seijihg) [inline api/cases/generated_documents/tests/test_generate_document.py:538]: @depsiatwal this cover it

**Review by depsiatwal — APPROVED** [review]
lgtm

File: api/cases/generated_documents/signing.py (hunk 1 above)
Comment (seijihg) [reply api/cases/generated_documents/signing.py]: ah yea :P thanks

**Pull request merged**